ffs/scheduler/internal/astore/astore.go

```diff
@@ -22,12 +22,13 @@ var (
 // Action represents an action to be executed by the Scheduler.
 type Action struct {
 	APIID       ffs.APIID
-	Cfg         ffs.CidConfig
+	Cid         cid.Cid
+	Cfg         ffs.StorageConfig
 	ReplacedCid cid.Cid
 }
 
 // Store is a Datastore backed implementation of ActionStore, which saves latests
-// PushConfig actions for a Cid.
+// PushStorageConfig actions for a Cid.
```

Comment:
This store as well is another place to potential migrate data. Same possible solutions as mentioned elsewhere.